plugins/package-managers/gradle-inspector/src/main/kotlin/GradleInspector.kt

```diff
@@ -350,14 +350,11 @@ private fun Collection<OrtDependency>.toPackageRefs(
     packageDependencies: MutableCollection<OrtDependency>
 ): Set<PackageReference> =
     mapTo(mutableSetOf()) { dep ->
-        val (id, linkage) = if (dep.localPath != null) {
-            val id = Identifier("Gradle", dep.groupId, dep.artifactId, dep.version)
-            id to PackageLinkage.PROJECT_DYNAMIC
+        val id = Identifier(dep.dependencyType, dep.groupId, dep.artifactId, dep.version)
```

Comment:
out of scope: 

I just realized that the inspector returns `Gradle` as `project.id.type`.
Would it make sense to either update the readme which you already touched in a previous PR, to  
not say anymore "not recommended" to "must not be enabled in parallel" as it would create ID clashes?

Reply:
I'll probably do that as part of my other PR. This is all prerequisite work to solve some issues with making GI the default anyway.